exporting/README.md

```diff
@@ -46,7 +151,132 @@ destination = localhost:4243
+    matching the hostname will be used - positive or negative).
+
+-   `send charts matching = *` includes one or more space separated patterns, using `*` as wildcard (any number of times
+    within each pattern). The patterns are checked against both chart id and chart name. A pattern starting with `!`
+    gives a negative match. So to match all charts named `apps.*` except charts ending in `*reads`, use `!*reads
+    apps.*` (so, the order is important: the first pattern matching the chart id or the chart name will be used -
+    positive or negative).
+
+-   `send names instead of ids = yes | no` controls the metric names Netdata should send to the external database.
+    Netdata supports names and IDs for charts and dimensions. Usually IDs are unique identifiers as read by the system
+    and names are human friendly labels (also unique). Most charts and metrics have the same ID and name, but in several
+    cases they are different: disks with device-mapper, interrupts, QoS classes, statsd synthetic charts, etc.
+
+-   `send configured labels = yes | no` controls if labels defined in the `[host labels]` section in `netdata.conf`
+    should be sent to the external database
+
+-   `send automatic labels = yes | no` controls if automatially created labels, like `_os_name` or `_architecture`
+    should be sent to the external database
+
+> Starting from Netdata v1.20 the host tags (defined in the `[backend]` section of `netdata.conf`) are parsed in
+> accordance with a configured backend type and stored as host labels so that they can be reused in API responses and
+> exporting connectors. The parsing is supported for graphite, json, opentsdb, and prometheus (default) backend types.
+> You can check how the host tags were parsed using the /api/v1/info API call. But, keep in mind that backends subsystem
+> is deprecated and will be deleted soon. Please move your existing tags to the `[host labels]` section.
+
+## monitoring operation
+
+Netdata provides 5 charts:
+
+1.  **Buffered metrics**, the number of metrics Netdata added to the buffer for dispatching them to the
+    external database server.
+
+2.  **Exporting data size**, the amount of data (in KB) Netdata added the buffer.
+
+3.  **Exporting operations**, the number of operations performed by Netdata.
+
+4.  **Exporting thread CPU usage**, the CPU resources consumed by the Netdata thread, that is responsible for sending the
+    metrics to the external database server.
+
+![image](https://cloud.githubusercontent.com/assets/2662304/20463536/eb196084-af3d-11e6-8ee5-ddbd3b4d8449.png)
+
+## alarms
+
+Netdata adds 4 alarms:
+
+1.  `exporting_last_buffering`, number of seconds since the last successful buffering of exported data
+2.  `exporting_metrics_sent`, percentage of metrics sent to the external database server
+3.  `exporting_metrics_lost`, number of metrics lost due to repeating failures to contact the external database server
+
```

Comment:
what is the 4th alarm?

Reply:
It was commented out because it never worked correctly. I deleted it from the doc. Fixed the number.